lib/rules/keyframe-selector-notation/index.mjs

```diff
@@ -107,20 +103,18 @@ const rule = (primary, _, context) => {
 					if (expFunc(selector, selectorsInBlock)) return;
 
 					const fixedSelector = fixFunc(selector);
-
-					if (context.fix) {
-						fixer(fixedSelector);
-
-						return;
-					}
+					const index = tag.sourceIndex;
+					const endIndex = index + selector.length;
 
 					report({
 						message: messages.expected,
 						messageArgs: [selector, fixedSelector],
 						node: keyframeRule,
 						result,
 						ruleName,
-						word: selector,
```

Comment:
Should we support `word` in `isFixApplied`?
i.e. instead of requiring `index`/`endIndex`